src/mutators/RemoveConditionalsMutator.ts

```diff
@@ -1,43 +1,41 @@
-import {Syntax} from 'esprima';
-import {Mutator} from 'stryker-api/mutant';
+import { Syntax } from 'esprima';
+import { Mutator } from 'stryker-api/mutant';
+import * as estree from 'estree';
+
+type ConditionExpression = estree.DoWhileStatement | estree.IfStatement | estree.ForStatement | estree.WhileStatement | estree.ConditionalExpression;
 
 /**
  * Represents a mutator which can remove the conditional clause from statements.
  */
 export default class RemoveConditionalsMutator implements Mutator {
   name = 'RemoveConditionals';
-  private types = [Syntax.DoWhileStatement, Syntax.IfStatement, Syntax.ForStatement, Syntax.WhileStatement];
+  private types: string[] = [Syntax.DoWhileStatement, Syntax.IfStatement, Syntax.ForStatement, Syntax.WhileStatement, Syntax.ConditionalExpression];
 
   constructor() { }
 
-  applyMutations(node: ESTree.Node, copy: (obj: any, deep?: boolean) => any): ESTree.Node[] {
-    let nodes: ESTree.Node[] = [];
-
+  applyMutations(node: estree.Node, copy: <T>(obj: T, deep?: boolean) => T): estree.Node[] {
     if (this.canMutate(node)) {
-      let mutatedFalseNode: ESTree.Literal = copy((<ESTree.ConditionalExpression>node).test);
-      this.mutateTestExpression(mutatedFalseNode, false);
-      nodes.push(mutatedFalseNode);
+      let nodes: estree.Node[] = [];
+      nodes.push(this.booleanLiteralNode(node.nodeID, false));
```

Comment:
This is in fact incorrect. We should not replace the current node with a `false`/`true` but the test expression.